apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.ejs

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<%- include('../partials/head'); %>
+</head>
+<body>
+	<script>
+		$(function () {
+			$('[data-toggle="popover"]').popover({ sanitize: false })
+			$('#add_new_override').on('submit', function (event) {
+				var startDate = new Date($('#start_date').val());
+				var endDate = new Date($('#end_date').val());
+				if (startDate >= endDate) {
+					event.preventDefault();
+					$('#end_date').addClass('is-invalid');
+					$('#end_date_error').text('End date should be greater than the start date.').show();
+				}
+				// Check if either student_uid or group_id is not empty
+				if (userId === '' && groupName === '') {
+					event.preventDefault();
+					$('#group_id').addClass('is-invalid');
+					$('#group_id_error').text('Either user ID or group ID should be filled.').show();
+				}
+				
```

Comment:
A few things to check here:

- AFAICT, there is no element with id `group_id_error`
- Is this going to show a sensible error message on both group and non-group assessments?

In general, I'd recommend testing this for yourself by leaving the group name / student UID fields blank, trying to submit, and seeing that happens.

Frankly, you could probably get away with just adding the `required` attribute to these inputs and deleting this JS + corresponding `#..._error` display elements. The browser is very good at enforcing required fields with `required`.

Reply:
Fixed in 
65cbf40 (NOTE: I also assume credit is required)